### PR TITLE
readme: update web hosting instructions

### PR DESCRIPTION
**Pull request by jenlij**

### CODE REVIEW

**Review by egonelbre — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,36 +59,51 @@ After configuration is complete, running the link sharing is as simple as:
...
+    `Type TXT, Hostname txt-yourHostName, Content storj-access:3WXG1qE`
     
-4. That's it! You should be all set to share your files or directories with your custom domain.
+    <img src="docs/images/access.png" width="50%">
+
+4. You can check to make sure your dns records are ready with `dig @1.1.1.1 txt-<yourHostName>.<yourDomain> TXT`        
+    
+5. Without further action, your site will be served with http. You can secure your site by using a https proxy server such as [Cloudflare](https://www.cloudflare.com/)
+
+6. That's it! You should be all set to access your website e.g. `http://yourHostName.yourwebsite.com`
```
Comment (egonelbre) [inline README.md:107]: Use either `example.com` or `website.test`. `yourwebsite.com` is a public website that does not belong to storj.

**Review by egonelbre — CHANGES_REQUESTED** [review]

**Review by egonelbre — APPROVED** [review]

**Review by egonelbre — DISMISSED** [review]
the url-s need fixing

**Review by jtolio — APPROVED** [review]
we should wait to submit until we make sure the production auth service is stable and will reliably store access keys (we might still wipe the db)

**Comment by jenlij** [comment]
> we should wait to submit until we make sure the production auth service is stable and will reliably store access keys (we might still wipe the db)

Do you know when the prod auth service would be stable? @jtolio

File: README.md [hunk 2]
```diff
@@ -59,36 +59,65 @@ After configuration is complete, running the link sharing is as simple as:
 $ linksharing run
 ```
 
-## Custom URL configuration and static site hosting*
+## Standard Linksharing with Uplink
+Anything shared with `--url` will be readonly and available publicly (no secret key needed).
```
Comment (jtolio) [inline README.md:63]: unless you say --readonly=false, right?

Comment (jenlij) [reply README.md:63]: We only allow url and dns creation `if len(shareCfg.AllowedPathPrefix) == 1 && !permission.AllowUpload && !permission.AllowDelete`

Comment (jenlij) [reply README.md:63]: And unless I'm missing something, we don't render the page on the linksharing side unless public and readonly as well (or at least only GET requests are permitted)

Comment (jtolio) [reply README.md:63]: i think the access key doesn't have to be readonly for the linksharing side to work, but it does have to be registered as public.

but okay your other comments answer my question!

**Review by jtolio — APPROVED** [review]

**Review by cam-a — APPROVED** [review]

**Pull request merged**